apps/web/src/pages/ProfileAndApplicationsPage/components/ProfileAndApplicationsHeading.tsx

```diff
@@ -147,8 +147,8 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
       subtitle={intl.formatMessage(
         {
           defaultMessage:
-            "Manage your <a1>profile</a1>, <a2>career timeline</a2>, <a3>skills</a3>, and <a4>track applications</a4>.",
-          id: "zTJ2DT",
+            "Manage your <a1>personal info</a1>, <a2>career timeline</a2>, <a3>skills</a3>, and <a4>track applications</a4>.",
```

Comment:
I'm assuming the informal "info" is an acceptable use for our application language? Or, would it be better to use the full "information"?

I guess it is more of a question for the copy meeting, are we going to prefer informal over formal English? It just seems odd to mix them with no specific reason.

Reply:
@esizer 

I'm also not really concerned about this particular instance of a shortened word - I wouldn't say "info" is overly informal these days, though if we absolutely have to choose one or the other with good reason, we should probably stick with information. I doubt this is the only place this occurs on the site, and it's possible other informal variants of words also exist.

Reply:
Let's go with "information"